app/protected/route.js

```diff
@@ -39,12 +39,16 @@ export default Route.extend(AuthenticatedRouteMixin, {
     const usermodel = await this.store.peekRecord("user", user.data.id);
 
     // Fetch current employment
-    await this.store.query("employment", {
+    const employment = await this.store.query("employment", {
       user: usermodel.id,
       date: moment().format("YYYY-MM-DD"),
       include: "location"
     });
 
+    if (!employment.length) {
```

Comment:
In case of no employment `employment.length` is false?

Reply:
yes

Reply:
I mean it's `0` in that case, which equals to false